src/app/page/Status/Status.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import { useFirestore, firestoreConnect } from "react-redux-firebase";
+import { useSelector, connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import { getFirebase, withFirestore } from "react-redux-firebase";
+// objects
+import { User } from "../../model";
+// styles
+import { Typography, Button } from "@material-ui/core";
+import { Page } from "../../layout";
+import styled from "styled-components";
+
+const StyledHeader = styled(Typography)`
+  margin-top: 12vh;
+  padding: 0 3vw;
+`;
+
+const StyledCopy = styled(Typography)`
+  margin-top: 3vh;
+`;
+
+const StyledP = styled(Typography)`
+  margin-top: 3vh;
+`;
+
+const StyledButton = styled(Button)`
+  margin-top: 24px;
+  flex-grow: 1;
+  width: 20vw;
+`;
+
+const Status = ({ firestore }) => {
+  // a happy little message
+  const [Thank, setThank] = useState(false);
```

Comment:
minor: rename Thank to thank just to be consistent with variable naming. Not a blocker obviously.